runtime/jcl/jcl_java.tdf

```diff
@@ -0,0 +1,28 @@
+// Copyright IBM Corp. and others 2025
+//
+// This program and the accompanying materials are made available under
+// the terms of the Eclipse Public License 2.0 which accompanies this
+// distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+// or the Apache License, Version 2.0 which accompanies this distribution and
+// is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// This Source Code may also be made available under the following
+// Secondary Licenses when the conditions for such availability set
+// forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+// General Public License, version 2 with the GNU Classpath
+// Exception [1] and GNU General Public License, version 2 with the
+// OpenJDK Assembly Exception [2].
+//
+// [1] https://www.gnu.org/software/classpath/license.html
+// [2] https://openjdk.org/legal/assembly-exception.html
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+
+Executable=jcl_java
+DATFileName=JCLTraceFormat.dat
```

Comment:
I don't think we need or want to use a new `DATFileName`; using new `Executable` names is sufficient.
Otherwise several other places would need to change to accommodate the new file.
* debugtools/DDR_VM/src/com/ibm/j9ddr/tools/ddrinteractive/commands/SnapFormatCommand.java
* jcl/src/openj9.dtfj/share/classes/com/ibm/jvm/j9/dump/extract/Main.java
* jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/TraceFormat.java
* jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/traceformat/TraceFormat.java
* test/functional/RasapiTest/src/com/ibm/dump/tests/version_info/CheckVersionString.java